ballerina/tests/resources/expected_results/fragments_with_input_variables.json

```
@@ -0,0 +1,9 @@
+{
+    "data": {
+        "profile": {
+            "address": {
+                "city": "Albuquerque"
+            }
+        }
+    }
+}
```

Check the other places as well.
```suggestion
}

```